h/security.py

```diff
@@ -34,18 +31,18 @@ def derive_key(key_material, salt, info):
     """
     Derive a fixed-size (64-byte) key for use in cryptographic operations.
 
-    See https://cryptography.io/en/latest/hazmat/primitives/key-derivation-functions/
+    The key is derived using HKDF with the SHA-512 hash function. See
+    https://tools.ietf.org/html/rfc5869.
 
     :type key_material: str or bytes
     :type salt: bytes
     :type info: bytes
     """
     if not isinstance(key_material, bytes):
         key_material = key_material.encode()
-    algorithm = hashes.SHA512()
-    length = algorithm.digest_size
-    hkdf = HKDF(algorithm, length, salt, info, backend)
-    return hkdf.derive(key_material)
+
+    pseudorandom_key = hkdf_extract(salt, key_material, hash=hashlib.sha512)
+    return hkdf_expand(pseudorandom_key, info, length=64, hash=hashlib.sha512)
```

Comment:
I appreciate the variable name `pseudorandom_key` (rather than `prk` or something)